src/rust/ensogl/lib/theme/src/lib.rs

```diff
@@ -94,10 +94,25 @@ macro_rules! define_theme {
                 app.themes.register(stringify!($name),$name);
                 app.themes.set_enabled(&[stringify!($name)]);
             }
+
+            /// Enables current theme.
+            pub fn enable(app:&Application) {
+                let $name = theme::Theme::new();
+                app.themes.set_enabled(&[stringify!($name)]);
+            }
```

Comment:
My previous comment is not fully applied here. But let the linter do the job instead of me here - please check why linter is not checking this code at all. It should gave error on unused `mut` earlier, but did not (CI was happy). It sohuld give warning now too.